iac-extensions/terraform/src/main/java/org/sonar/iac/terraform/checks/gcp/PublicAccessCheck.java

```diff
@@ -48,19 +48,23 @@
   protected void registerResourceConsumer() {
     register(IAM_BINDING_RESOURCES,
       resource -> resource.list("members")
-        .reportItemIf(matchesPattern(".*(?:allUsers|allAuthenticatedUsers).*"), MESSAGE));
+        .reportItemIf(matchesPattern(".*all(Authenticated)?Users.*"), MESSAGE));
 
     register(IAM_MEMBER_RESOURCES,
       resource -> resource.attribute("member")
-        .reportIf(matchesPattern(".*(?:allUsers|allAuthenticatedUsers).*"), MESSAGE));
+        .reportIf(matchesPattern(".*all(Authenticated)?Users.*"), MESSAGE));
 
     register(Set.of("google_storage_default_object_access_control", "google_storage_object_access_control"),
       resource -> resource.attribute("entity")
-        .reportIf(matchesPattern("allUsers|allAuthenticatedUsers"), MESSAGE));
+        .reportIf(matchesPattern("all(Authenticated)?Users"), MESSAGE));
 
     register("google_bigquery_dataset_access",
       resource -> resource.attribute("special_group")
-        .reportIf(matchesPattern("allUsers|allAuthenticatedUsers"), MESSAGE));
+        .reportIf(matchesPattern("all(Authenticated)?Users"), MESSAGE));
+
+    register("google_storage_bucket_acl",
```

Comment:
What I see is SONARIAC-365 specifies not 1 but 3 relevant resource types here:
1. google_storage_bucket_acl
2. google_storage_default_object_acl
3. google_storage_object_acl

Why is only the first one of those handled?

Reply:
Nice catch